librz/core/cmd_api.c

```diff
@@ -1684,8 +1765,7 @@ RZ_API char *rz_cmd_parsed_args_execstr(RzCmdParsedArgs *a) {
 }
 
 RZ_API const char *rz_cmd_parsed_args_cmd(RzCmdParsedArgs *a) {
-	rz_return_val_if_fail(a && a->argv && a->argv[0], NULL);
-	return a->argv[0];
+	return a && a->argv && a->argv[0] ? a->argv[0] : NULL;
```

Comment:
why this change?

Reply:
rz_return_val_if_fail is a macro that can be skipped if the right flags are defined (meant for debugging).

Reply:
I know, indeed I assume that the condition in `rz_return_val_if_fail` is always true. Indeed `RzCmdParsedArgs` is created through `rz_cmd_parsed_args_new`, which always allocates `a->argv` and always puts a value in `a->argv[0]`. So I think the condition should stay how it was before and it can never be false. If it is, there's some error somewhere else.